hash.go

```diff
@@ -28,8 +28,6 @@ import (
 	"unsafe"
 )
 
-func _hash(digests *byte, p [][32]byte, count uint32)
```

Comment:
Why move this?

Reply:
To make space for an empty implementation in the hash_unsupported. Otherwise there would be two declarations of this function.